resources/entity/player/Player.gd

```diff
@@ -109,10 +114,6 @@ func set_shader_value(value: float):
 	$SubViewportContainer/SubViewport/AnimatedSprite2D.material.set_shader_parameter("flash_modifier", value)
 
 
-func add_health(amount: int):
-	$HealthComponent.hp += amount
```

Comment:
Might be better to leave this in - because of emitting signals for the HUD (potentially)